index.js

```diff
@@ -0,0 +1,87 @@
+module.exports = {
+  parser: "@typescript-eslint/parser",
+
+  extends: ["plugin:react/recommended", "prettier/@typescript-eslint", "plugin:prettier/recommended", "prettier/react"],
+
+  plugins: ["simple-import-sort", "jest", "@typescript-eslint", "react-hooks", "react", "prettier", "import"],
+
+  ignorePatterns: ["coverage", "**/dist", ".cache-loader", "node_modules", ".yalc"],
+
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: "module",
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+
+  settings: {
+    react: {
+      version: "detect",
+    },
+  },
+
+  env: {
+    browser: true,
+    node: true,
+    jest: true,
+    es6: true,
+  },
+
+  rules: {
+    "prettier/prettier": "error",
+
+    // Base
+
+    "no-unused-expressions": ["error", { allowTernary: true }],
+    "no-console": "off",
+    "no-confusing-arrow": "off",
+    "no-else-return": "off",
+    "no-return-assign": ["error", "except-parens"],
+    "no-underscore-dangle": "off",
+    camelcase: "off",
+    "prefer-arrow-callback": ["error", { allowNamedFunctions: true }],
+    "class-methods-use-this": "off",
+    "no-restricted-syntax": "off",
+    "no-param-reassign": ["error", { props: false }],
+    "arrow-body-style": "off",
+
+    // Imports
+
+    "simple-import-sort/sort": "error",
+    "sort-imports": "off",
```

Comment:
Do we really need that `single-import-sort` thingy? I always used `import` and it never disappointed me.
I'm always all in for keeping the list of plugins as small as possible so that it's easier to upgrade ESLint in future + the risk of changes is rather lower too.

Reply:
Yeah, it is really good - I tried with and without.